formats/person.jsonnet

```diff
@@ -4,6 +4,19 @@
       type: "object",
       additionalProperties: false,
       properties: {
+        analytics_identifier: {
+          "$ref": "#/definitions/analytics_identifier",
+        },
+        full_name: {
+          type: "string",
+          description: "Name of the person, including titles and any letters, eg: 'Sir Lord Snoopy DOG'",
+        },
+        minister: {
+          type: "boolean"
```

Comment:
Probably best to add a `,` at the end for consistency

Reply:
I've addressed this, thank you